src/measure_theory/function/lp_space.lean

```diff
@@ -2069,7 +2069,7 @@ def comp_Lpₗ (L : E →L[𝕜] F) : (Lp E p μ) →ₗ[𝕜] (Lp F p μ) :=
     ext1,
     filter_upwards [Lp.coe_fn_smul c f, coe_fn_comp_Lp L (c • f), Lp.coe_fn_smul c (L.comp_Lp f),
       coe_fn_comp_Lp L f] with _ ha1 ha2 ha3 ha4,
-    simp only [ha1, ha2, ha3, ha4, map_smul, pi.smul_apply],
+    simp only [ha1, ha2, ha3, ha4, continuous_linear_map.map_smul, pi.smul_apply],
```

Comment:
Do you have a suggestion why `map_smul` work unqualified here? `continuous_linear_map` has a `continuous_linear_map_class` instance right?

Reply:
Actually it does work -- you just have to spell it `smul_hom_class.map_smul` to distinguish it from `filter.map_smul`.

Reply:
I see, we should probably make `filter.map_smul` protected. But for now:
```suggestion
    simp only [ha1, ha2, ha3, ha4, smul_hom_class.map_smul, pi.smul_apply],
```